### DISCUSSION TITLE
Frontend counter - DONE

### DISCUSSION THREAD

**Discussion by zu** [ibericode/koko-analytics#238] [Ideas]
Make it possible to show total visitors and visitors per day, month or year on the frontend.
For the total number of visitors, it should be possible to enter a start number to continue from / replace a different visitor counter.

**Comment by dannyvankooten** [comment]
As of 893cf5ceb58664aae1d196dd3aab8f9afcf07dc0, there is a shortcode for this.

To show the total # of visitors for the current page over the last 10 years:

```
[koko_analytics_counter]
```

To show the total # of pageviews for the entire site over the last 30 days:

```
[koko_analytics_counter days="30" metric="pageviews" global="true"]
```


To show the total # of pageviews for the current page over the last 30 days:

```
[koko_analytics_counter days="30" metric="pageviews"]
```
